modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraExperimentRepository.java

```diff
@@ -353,33 +360,18 @@ protected Experiment internalGetExperiment(Application.Name appName,
      */
     @Override
     public Experiment.ID createExperiment(NewExperiment newExperiment) {
-
-        validator.validateNewExperiment(newExperiment);
-
-        // Ensure no other experiment is using the label
```

Comment:
@abhishekjain88 Yes, that is being checked in ExperimentsImpl.createExperiment() method...